### ISSUE TITLE
Zoom drop-down doesn't work on About page

### ISSUE THREAD

**Issue by ChasonMindImp** [maltyeva/mindimp-pg#51]
When on https://app.mindimp.com/about, the Zoom link for the drop down does not work.

**Comment by MindImpMatt** [comment]
I can't get the Zoom drop down link to work on most non-home pages.

**Comment by maltyeva** [comment]
yeah, thats a javascript issue that I'm having trouble resolving. hoping the sidebar will take care of some of that.

**Comment by ChasonMindImp** [comment]
I am having a similar issue with the sidebar. For example, from https://app.mindimp.com/articles/20# I am unable to navigate to Articles using the link in the sidebar.

**Comment by maltyeva** [comment]
hey guys, 

i made some changes to the javascript, which should hopefully resolve these issues, but it needs more testing. please keep an eye on this in the next few days and let me know if there are still issues
